readthedocs/proxito/views/serve.py

```diff
@@ -226,10 +226,11 @@ def get(self, request, proxito_path, template_name='404.html'):
                 )
                 # Use urlparse so that we maintain GET args in our redirect
                 parts = urlparse(proxito_path)
+                path = parts.path.rstrip('/')
```

Comment:
Can we name this different than just `path`? We use the same name later in the this function as well. Would be good if it can communicates that it does not have the trailing slash in its name.